test.js

```diff
@@ -74,6 +74,33 @@ test('excludes and includes options', function (t) {
 	t.end();
 });
 
+test('shortForms options', function (t) {
+	var actual = dargs({a: 'foo', file: 'test'}, {shortForms: {file: 'f'}});
+	var expected = [
+		'--a=foo',
```

Comment:
Implicitly causing any single letter arguments to use a single "-" makes sense, but I think should be a separate pull request. There is actually an [existing closed pull request](https://github.com/sindresorhus/dargs/pull/4) that does just that. 

This is focused on making the conversion to single "-" arguments explicit so that the end user can handle cases where only the single "-" argument is available, but still expose a readable `options` input to be handled by `dargs`.